templates/CommentsInterface_singlecomment.ss

```diff
@@ -1,31 +1,37 @@
-<div class="comment" id="<% if isPreview %>comment-preview<% else %>$Permalink<% end_if %>">
-<% if $Gravatar %><img class="gravatar" src="$Gravatar" alt="Gravatar for $Name" title="Gravatar for $Name" /><% end_if %>
-	$EscapedComment
-</div>
+<div class="comment" data-comment-id="$ID" id="<% if isPreview %>comment-preview<% else %>$Permalink<% end_if %>">
+<% if MarkedAsDeleted %><% _t('DELETED_BY_ADMIN','This comment was deleted by an administrator') %><% else %>
+<% if $Gravatar %><div class="gravatarContainer"><img class="gravatar" src="$Gravatar" alt="Gravatar for $Name" title="Gravatar for $Name" /></div><% end_if %>
+<div class="actualcomment">
```

Comment:
Naming is a bit... on the nose, don't you think? :P Not that I can blame you much.